Orleans.Providers.MongoDB/MongoDBSiloExtensions.cs

```diff
@@ -51,6 +51,7 @@ public static ISiloBuilder UseMongoDBClient(this ISiloBuilder builder, Func<ISer
         public static ISiloBuilder UseMongoDBReminders(this ISiloBuilder builder,
                 Action<MongoDBRemindersOptions> configurator = null)
         {
+            builder.AddReminders();
```

Comment:
I think this call better belongs in `AddMongoDBReminders` methods so that it covers all use cases.

Reply:
Done